### ISSUE TITLE
I WAS SO WRONG, I DIDN'T KNOW ABOUT THE ARCHIVES TAB, TELL Ousnius I'M SO SORRY. DEVS & OBJ skip to What Happen so Far 2.

### ISSUE THREAD

**Issue by Lukong15** [niftools/nifskope#108]
Today I discovered in order to load Textures and Materials for NIFs in Nifskope was to Render > "Archives" Tab ...I thought adding "Data" Folder alone was good enough and that the reason it wasn't working was due to the program in Alpha & Not having Fallout 4 Support (So I said in my foolishness I thought Nifskope was obsolete cause of no OBJ Support & Viewable Textures & that Bodyslide with certain features would be better). 
Now I can quickly open NIFs and see them with Textures & Materials now (Good for Resource Gathering...if you want picks I'll attach). _Okay onto the next topic._

## Dear NIFSkope, 3DSMax NIF & Blender NIF Developers
A few features I asked Ousnius for were denied. What I discovered is I was able to get a NIF to Blender and back into Game without 3DSmax and without UV being Destroy and only needing to restore Texture Paths & Material Paths for NIFs (Non-Complex ones), but if the NIF had Hierarchy I can use an "old" Format via 3DSMax though the ObjectNames were "shorten" (_I haven't tested with what I later discovered and what I did before...otherwise I think I can Convert NIF to keep Heirarchy, UV Map & only need to restore ObjectNames & Material Path_).

- Attempts
At first I was trying to find the right Conversion Combo (2 days, 13hrs total) so I can edit NIF in Blender and then load into game. "NIF to 3DSMax > Blender > 3DSMax > Game" worked, but converting NIF using "NIF to Bodyslide > Blender > Bodyslide" = Game Crashed when I got close to the Mesh in-game), but the 3DSMax Conversion Combo showed invisible Mesh (Cause I couldn't see it, but it didn't crash).
When I compared the Converted with Vanilla NIF and attempt to edit in NIFskope to add the missing Materials & Textures path from Vanilla NIF to Converted NIF = Game Crashed when I loaded the Game.
I also tested,  Converted Edited NIF & Converted Vanilla NIF that where unedited so I knew it had nothing to do with editing.

- Fix Invisible/Purple Textured Mesh
It was in Bodyslide that I found where & why the Mesh was invisible because I wondered why Bodyslide knew where to look for Texture & Material even though both Bodyslide & NifSkope (at the time) both pointed at the Data Folder and Blender & 3DSMax needed Textures in same folder. 
So I opened & compared a Vanilla NIF to My Converted/Edited NIF > curious what "**Properties**" was > They where different (Filled vs Blank)... each object in a NIF needed Materials & Texture Path assigned to it.
- - When I request a Batch version of this from Ousnius, I was told "Its a specific use case" literally with Bodyslide I found a way to fix both Crashing and Invisible/Purple Mesh NIF, but its so HARD TRYING TO FIX THAT FOR 30 Pieces of a...1 piece vanilla NIF I modularize for customization so I ended up having to learn how to assign Materials in 3DSMax -which had Batch Assigning- and later found out I needed 3DSMax to avoid problems that Bodyslide was causing...I don't know why

- Fix Crashing
So after figuring out how to make Converted NIF to appear in Fallout 4 as I'm also making a Tutorial with  Thorough Steps so others can make something similar to my Mod (as when I tried to make my mod I was discouraged by the lack of quality or outdated Tutorials and gave up 3-5 times). 
I changed the Conversion Combo to "NIF > 3DSMax > Blender > Bodyslide (For Materials & Texture Assigning) > 3DSMax > Game"...I got curious and wondered if I can skip 3DSMax entirely (So don't have to worry about buying 3DSMax)...GAME CRASHED...so I checked to see if it was something in the Properties...Compared Vanilla to Converted...and unchecked "something" (You've probably got it by now)

## What happen so Far
0. So since while trying to request features from Ousnius I upload a video of my progress of the Mod (cause I didn't come to Ousnius without having any work to show him/her)...and now I've pass the FO4Edit & Records Stage, but then I fill into more problems, but now I have 2 ways of solving it (Removing Doubles or Lessen the Limited Modularization by combining Parts...I hope that is convoluted though Ousnius could probably tell you what I'm attempting)

1. Bugs in Bodyslide when I put Torso PA Mod (Clue) with a certain "object" and the Nifskope can see it fine, but Bodyslide & Game see it stretching (Only when I remove the Torso is the PA Mod fine). 
So I use 3DSMax (Which I realize you are the people working on Blender NIFscript...I want to make a good enough initial Mod that can get the ball rolling for donations to swing your way in exchange for Fallout 4 NIF Blender Support, Material Batch Editing, Copy ObjectNames, Materials, Texture or Hierarchy from Original NIF File & Animation Support so I can make the rest of my Mod Project easily & Quickly)
**Reasons for Blender Support**
1a. **Material Batch Editing** = Blender has this, but in case like Texture Paths, etc
1b. **Copy ObjectNames from Original NIF** = If I Export an Object from NIF with Multiple Objects and it has a Colon it, Windows will turn up a 0KB file cause Colon isn't allowed...but get this there is an old Format that can keep Hierarchy(I know) & UV(I think), but loses the ObjectNames cause it shortens them...3DS, but in doing so I see 3DS keeps a similar name so if there was a Script or Function that could compare names that really close and Give them back there name that would allow people to Keep UV & Hierarchy or at least have you own Metadata so whatever format is losing something like FBX = UV Map. OBJ = Materials/Textures/Heirarchy or 3DS = Names...that the metadata can restore the NIF
1c. **Copy Hierarchy, Materials &Texture from Original NIF** = This was cause I was attempting to do another Mod (Making the Pipboy...a BigLady...Do you want Screenshots?) and found out some NIF have many Objects with some using Materials, others used Textures without Materials and both Bodyslide & 3DSMax NIF can destroy the Hierarchy or positioning sometimes of the Objects (Hence why I thought the PipBoy was disappearing, but could later see it in pieces)
1d. **Animation Support** = I'm a Lvl 38 Tutorial BornCG  Blender (LOVE THIS GUYS VIDEOS HE CLEAR AND WELL SPOKEN and TAKES HIS TIME TO TEACH)...so really if we get NIF support...no only really has to buy 3DSMax for 4000 (Mines expires in a Year)

2. About the Request: **OBJ DOESN'T DESTROY NIFs COMPLETELY** DEPENDING ON WHAT YOUR MODDING...I FOUND A PROBLEM THAT COULD EASILY FIX OBJ to NIF in Bodyslide, if you can find the script in Bodyslide I tell what I did, otherwise I'm terribly sorry for wasting your time if you've already fix it

3. I have to go somewhere...sorry this post is incomplete

**Comment by neomonkeus** [comment]
Closing as completely incomprehensible. I have attempted to read through your previous issue post.

- Github issues are not meant to be forum style posts/stories. 
- This repo is strictly for NifSkope issues only, 
- keep things concise, describe the precise issue you have, one per issue.
- We won't be support .obj format.
- Blender issues go to the Blender Nif Plugin repo, You are describing a workflow, not issues you currently have. There are already issues which address point those workflows.
